pkg/cluster/ansible/config.go

```diff
@@ -42,7 +43,7 @@ func ImportConfig(name string, clsMeta *spec.ClusterMeta, sshTimeout int64, nati
 					SSHKeySet(
 						spec.ClusterPath(name, "ssh", "id_rsa"),
 						spec.ClusterPath(name, "ssh", "id_rsa.pub")).
-					UserSSH(inst.GetHost(), inst.GetSSHPort(), clsMeta.User, sshTimeout, nativeClient).
+					UserSSH(inst.GetHost(), inst.GetSSHPort(), clsMeta.User, sshTimeout, sshType, "").
```

Comment:
What if the `sshType` is empty?

Reply:
Then the default one (builtin) will be used

Reply:
Seems not according to current implementation.

Reply:
https://github.com/pingcap/tiup/pull/699/files#diff-b64eb542eb081f4f4515d3e180b8d16cR67

```
	if etype == "" {
		etype = SSHTypeBuiltin
	}
```